module/VuFind/src/VuFind/Db/Table/ResourceTags.php

```diff
@@ -561,7 +561,7 @@ public function getUniqueTags($userId = null, $resourceId = null, $tagId = null)
                 $select->where->equalTo('resource_tags.tag_id', $tagId);
             }
             $select->group(['tag_id', 'tag']);
-            $select->order([new Expression('lower(tag)')]);
+            $select->order([new Expression('lower(tag)'), 'tag']);
```

Comment:
This does not seem to help fixing the problem. Is it supposed to be included in the PR anyway?

Reply:
I just read the description of the PR again. That makes sense to me.

Reply:
Great! If tests pass for you now and you don't mind approving, I can get this merged. :-)